spec/system/candidate_interface/continuous_applications/submitting/candidate_submitting_application_spec.rb

```diff
@@ -28,6 +28,15 @@
     # and_i_receive_an_email_confirmation
   end
 
+  it 'Candidate with an application missing the science GCSE' do
+    given_i_am_signed_in
+
+    when_i_have_completed_my_application
```

Comment:
I'd renamed to   when_i_have_completed_my_application  but also   when_i_have_added_primary_as_course_choice

Reply:
I think we accidently added `'it's` we could rename to `scenario`